packages/node_modules/@ciscospark/plugin-phone/src/phone.js

```diff
@@ -192,8 +198,11 @@ const Phone = SparkPlugin.extend({
           if (!this.emittedCalls.get(makeInternalCallId(locus))) {
             const callItem = Call.make({locus, parent: this.spark});
             calls.push(callItem);
-            this.trigger('call:incoming', callItem);
             this.emittedCalls.add(callItem);
+            this.trigger(events.CALL_CREATED, callItem);
+            if (direction(locus) === 'in' && !joined(locus)) {
+              this.trigger(events.CALL_INCOMING, callItem);
```

Comment:
given that we've got the `call:created` event, I feel like this should probably just mirror the `onLocusEvent` logic, right?

Reply:
My concern was that the `alertType.action` might be not correct when doing a `locus.list` but it looks like it is correct after inspecting the network response. I'll have them use a shared function.